db/migrate/20231108210613_change_associations_duration_rates.rb

```diff
@@ -0,0 +1,17 @@
+class ChangeAssociationsDurationRates < ActiveRecord::Migration[7.0]
```

Comment:
Do we want to do something about existing DurationRates? At least destroy them all?

Reply:
💣  Yes let's start fresh